service/src/main/java/bio/terra/tanagra/app/configuration/AuthenticationConfiguration.java

```diff
@@ -96,6 +105,25 @@ public class AuthenticationConfiguration {
       environmentVariable = "TANAGRA_AUTH_GKE_BACKEND_SERVICE_ID")
   private String gkeBackendServiceId;
 
+  @AnnotatedField(
+      name = "tanagra.auth.unverifiedJwt.issuer",
+      markdown = "The issuer of JWT used for its verification. ",
+      environmentVariable = "TANAGRA_AUTH_UNVERIFIED_JWT_ISSUER")
+  private String unverifiedJwtIssuer;
```

Comment:
It's confusing to me to have configuration for verifying a JWT when the authentication method is called "unverifiedJwt". Perhaps you can treat this as a different authentication method (e.g. "auth0Jwt" or "auth0VerifiedJwt")?

Reply:
Autho library, but it can work for other issuers too. If we want to tailor it to only auth0, we can name it TANAGRA_AUTH_AUTH0_JWT
IapJwtUtils could be modified to be generic for all JWT. trying that if those can be renamed and refactored

Reply:
Renamed to TANAGRA_AUTH_JWT for generic JWT, refactored IapTokenUtils to share the JWT decoding